jwst/ramp_fitting/gls_fit.py

```diff
@@ -692,52 +681,44 @@ def gls_fit(ramp_data, input_var_data,
 
     # Divide by sqrt(2) to convert the readnoise from CDS to single readout.
     rn3d = readnoise.reshape((nz, 1, 1)) * SINGLE_READOUT_RN_FACTOR
```

Comment:
The factor of 1 / sqrt(2) has already been applied by function utils.get_ref_subs.

Reply:
Removed multiplication by SINGLE_READOUT_RN_FACTOR